client/src/core/client/admin/routes/Reports/ReportedComment.tsx

```diff
@@ -0,0 +1,198 @@
+  Timestamp,
+} from "coral-ui/components/v2";
+import { StarRating } from "coral-ui/components/v3";
+
+import { ReportedComment_dsaReport } from "coral-admin/__generated__/ReportedComment_dsaReport.graphql";
+
+import styles from "./ReportedComment.css";
+
+interface Props {
+  dsaReport: ReportedComment_dsaReport;
+  onShowUserDrawer: (userID?: string) => void;
+}
+
+const ReportedComment: FunctionComponent<Props> = ({
+  dsaReport,
+  onShowUserDrawer,
+}) => {
+  const { comment } = dsaReport;
+  const inReplyTo = comment && comment.parent && comment.parent.author;
+
+  // TODO: Maybe show something like Comment not available in this case
+  if (!comment) {
+    return null;
+  }
+
+  return (
+    <Flex direction="column">
+      <Flex direction="column" className={styles.commentMain}>
+        <Localized id="reports-singleReport-comment">
+          <div className={styles.label}>Comment</div>
+        </Localized>
+        <>
+          {comment.deleted ? (
+            <Localized id="comments-tombstone-deleted">
+              <div className={styles.data}>
+                This comment is no longer available. The commenter has deleted
+                their account.
+              </div>
+            </Localized>
+          ) : (
+            <>
+              <Flex className={styles.commentBox}>
+                <div>
+                  <div>
+                    <Flex alignItems="center">
+                      {comment.author?.username && (
+                        <>
+                          <UsernameButton
+                            className={styles.commentUsernameButton}
+                            username={comment.author.username}
+                            onClick={() => onShowUserDrawer(comment.author?.id)}
+                          />
+                          <CommentAuthorContainer comment={comment} />
+                        </>
+                      )}
+                      <Timestamp>{comment.createdAt}</Timestamp>
+                      {comment.editing.edited && (
+                        <Localized id="reports-singleReport-comment-edited">
+                          <span className={styles.edited}>(edited)</span>
+                        </Localized>
+                      )}
+                    </Flex>
+                    {inReplyTo && inReplyTo.username && (
+                      <InReplyTo
+                        className={styles.reportUsername}
+                        onUsernameClick={() => onShowUserDrawer(inReplyTo.id)}
+                      >
+                        {inReplyTo.username}
+                      </InReplyTo>
+                    )}
+                  </div>
+                  {comment.rating && (
+                    <div>
+                      <StarRating rating={comment.rating} />
+                    </div>
+                  )}
+                  <div>
+                    <div>
+                      {/* TODO: Do we want to show message that comment has been rejected? */}
```

Comment:
Just noting that we want to keep track of the various TODO's I'm seeing while reading. We should still merge this into the epic branch though.

Reply:
Yep! The design input on this is probably not, so I can actually probably delete this one now.